pygments/lexers/yara.py

```diff
@@ -0,0 +1,45 @@
+# https://github.com/infosec-intern/vscode-yara/blob/main/yara/syntaxes/yara.tmLanguage.json
+from pygments.lexer import RegexLexer, words
+#from pygments.util import get_bool_opt, shebang_matches
+from pygments.token import *
+
+__all__ = ['YaraLexer']
+
+class YaraLexer(RegexLexer):
+    """
+    For YARA rules
+    """
+
+    name = 'YARA'
+    url = 'https://virustotal.github.io/yara/'
+    aliases = ['yara', 'yar']
+    filenames = ['*.yar']
+
+    tokens = {
+        'root': [
+            (r'\s+', Text),
+            (r'//.*?$', Comment.Singleline),
+            (r'/\*', Comment.Multiline, 'comment'),
+            (words(('rule', 'private', 'global', 'import', 'include'), prefix=r'\b', suffix=r'\b'), Keyword.Declaration),
+            (words(('strings', 'condition', 'meta', 'import'), prefix=r'\b', suffix=r'\b'), Keyword),
+            (r'(true|false)\b', Keyword.Constant),
+            (r'(and|or|not)\b', Operator.Word),
+            (r'(any|all)\b', Operator.Word),
+            (r'(\$[\w\d]+)', Name.Variable),
+            (r'"[^"]*"', String.Double),
+            (r'\'[^\']*\'', String.Single),
+            #(r'0x(?:[0-9A-Fa-f?]{2}\s*)+(?:\[\d+-\d+\])?', Number.Hex),
+            #(r'0x\[[\da-fA-F?-]+\]', Number.Hex),
+            #(r'\d+(?:-[a-z\d]+)?', Number.Integer),
```

Comment:
I Think I fixed them. I removed the number regex because I think it is not needed.
Additionally, I changed the Hex regex, before it matched on anything between curly brackets, and that ruined the whole formatting of the example file. Now it only matches per line:
19bc5a8f804c3dd366fa297b9ce2c27dcf3986c1